service/controller/v3/network/network.go

```diff
@@ -0,0 +1,72 @@
+package network
+
+import (
+	"context"
+	"net"
+
+	"github.com/giantswarm/ipam"
+	"github.com/giantswarm/microerror"
+
+	"github.com/giantswarm/azure-operator/service/controller/setting"
+	"github.com/giantswarm/azure-operator/service/controller/v3/key"
+)
+
+const (
+	ipv4MaskSize = 32
```

Comment:
I think this should be 24

Reply:
No, this is the length in bytes of the CIDRMask which is 4*8 for IPv4